<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="WorkflowAuthorizationException.xml" source-language="en-US" target-language="es-ES">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac5ee8327669a02a2249fa26a696a9e644cb3832f0b.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">ee8327669a02a2249fa26a696a9e644cb3832f0b</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:System.Workflow.Activities.WorkflowAuthorizationException">
          <source>The exception that is thrown when role validation fails due to a specified identity that is not contained in the <ph id="ph1">&lt;see cref="T:System.Workflow.Activities.WorkflowRoleCollection" /&gt;</ph>.</source>
          <target state="translated">La excepción que se inicia cuando se producen errores de validación de funciones debido a que <ph id="ph1">&lt;see cref="T:System.Workflow.Activities.WorkflowRoleCollection" /&gt;</ph> no contiene una identidad especificada.</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" uid="T:System.Workflow.Activities.WorkflowAuthorizationException">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Workflow.Activities.WorkflowAuthorizationException" /&gt;</ph> class.</source>
          <target state="translated">Inicializa una nueva instancia de la clase <ph id="ph1">&lt;see cref="T:System.Workflow.Activities.WorkflowAuthorizationException" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" uid="M:System.Workflow.Activities.WorkflowAuthorizationException.#ctor">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Workflow.Activities.WorkflowAuthorizationException" /&gt;</ph> class.</source>
          <target state="translated">Inicializa una nueva instancia de la clase <ph id="ph1">&lt;see cref="T:System.Workflow.Activities.WorkflowAuthorizationException" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" uid="M:System.Workflow.Activities.WorkflowAuthorizationException.#ctor(System.String)">
          <source>The error message that explains the reason for the exception.</source>
          <target state="translated">Mensaje de error que explica el motivo de la excepción.</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" uid="M:System.Workflow.Activities.WorkflowAuthorizationException.#ctor(System.String)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Workflow.Activities.WorkflowAuthorizationException" /&gt;</ph> class using the specified message.</source>
          <target state="translated">Inicializa una nueva instancia de la clase <ph id="ph1">&lt;see cref="T:System.Workflow.Activities.WorkflowAuthorizationException" /&gt;</ph> usando el mensaje especificado.</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" uid="M:System.Workflow.Activities.WorkflowAuthorizationException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> that holds the serialized object data about the exception being thrown.</source>
          <target state="translated">
          <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> que contiene los datos serializados del objeto que hacen referencia a la excepción que se va a producir.</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" uid="M:System.Workflow.Activities.WorkflowAuthorizationException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.StreamingContext" /&gt;</ph> that contains contextual information about the source or destination.</source>
          <target state="translated"><ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.StreamingContext" /&gt;</ph> que contiene información contextual sobre el origen o el destino.</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" uid="M:System.Workflow.Activities.WorkflowAuthorizationException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Workflow.Activities.WorkflowAuthorizationException" /&gt;</ph> class by using serialized data.</source>
          <target state="translated">Inicializa una nueva instancia de la clase <ph id="ph1">&lt;see cref="T:System.Workflow.Activities.WorkflowAuthorizationException" /&gt;</ph> con datos serializados.</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" uid="M:System.Workflow.Activities.WorkflowAuthorizationException.#ctor(System.String,System.Exception)">
          <source>The error message that explains the reason for the exception.</source>
          <target state="translated">Mensaje de error que explica el motivo de la excepción.</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" uid="M:System.Workflow.Activities.WorkflowAuthorizationException.#ctor(System.String,System.Exception)">
          <source>The <ph id="ph1">&lt;see cref="T:System.Exception" /&gt;</ph> that caused the <ph id="ph2">&lt;see cref="T:System.Workflow.Activities.WorkflowAuthorizationException" /&gt;</ph>.</source>
          <target state="translated">La <ph id="ph1">&lt;see cref="T:System.Exception" /&gt;</ph> que produjo la <ph id="ph2">&lt;see cref="T:System.Workflow.Activities.WorkflowAuthorizationException" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" uid="M:System.Workflow.Activities.WorkflowAuthorizationException.#ctor(System.String,System.Exception)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Workflow.Activities.WorkflowAuthorizationException" /&gt;</ph> class by using the specified message and <ph id="ph2">&lt;see cref="T:System.Exception" /&gt;</ph>.</source>
          <target state="translated">Inicializa una nueva instancia de la clase <ph id="ph1">&lt;see cref="T:System.Workflow.Activities.WorkflowAuthorizationException" /&gt;</ph> con el mensaje especificado y <ph id="ph2">&lt;see cref="T:System.Exception" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" uid="M:System.Workflow.Activities.WorkflowAuthorizationException.#ctor(System.String,System.String)">
          <source>The name of the <ph id="ph1">&lt;see cref="T:System.Workflow.ComponentModel.Activity" /&gt;</ph> in which the exception occurred.</source>
          <target state="translated">Nombre de la <ph id="ph1">&lt;see cref="T:System.Workflow.ComponentModel.Activity" /&gt;</ph> donde se produce la excepción.</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" uid="M:System.Workflow.Activities.WorkflowAuthorizationException.#ctor(System.String,System.String)">
          <source>The name of the user or entity on whose behalf the workflow authorization exception was thrown.</source>
          <target state="translated">Nombre del usuario o entidad en cuyo el nombre que se inició la excepción de autorización de flujo de trabajo.</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" uid="M:System.Workflow.Activities.WorkflowAuthorizationException.#ctor(System.String,System.String)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Workflow.Activities.WorkflowAuthorizationException" /&gt;</ph> class using the name of the activity and the principal.</source>
          <target state="translated">Inicializa una nueva instancia de la clase <ph id="ph1">&lt;see cref="T:System.Workflow.Activities.WorkflowAuthorizationException" /&gt;</ph> con el nombre de la actividad y de la entidad de seguridad.</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>